docs/internal/Assignment-Process.md

```diff
@@ -13,18 +13,28 @@ First of all, some bulletpoint (and rules) to help clarify the diagram
 ### Assignment rules
 
 - One `review_assignment` should exist per reviewer/application/stage/level
-- Review assignment should only be editable by a reviewer if it's `Available for self-assignment`, otherwise by linked `Assigner`. Review assignment should not be editable once stage of review assignment is different to the current application stage.
+- Review assignment should only be assignable by a reviewer if has `is_self_assignable` = **True**, otherwise by linked `Assigner`.
+- Review assignment should not be editable once stage of review assignment is different to the current application stage.
 - `is_last_level` field is a helper field for front end (to avoid front ending needing to figure if if review is last level for given stage)
```

Comment:
By "helper field", do you mean it's auto-generated from other information? Maybe just say that explicitly? 

> `is_last_level` field is an automatically calculated field to help front-end....

Something like that?

Reply:
Yes, that's better. Thanks